R/dbConnection.R

```diff
@@ -12,8 +12,22 @@ connect <- function(host='localhost', username=NULL, password=NULL, database=NUL
 #' The metadata table. This is all of the different 
 #' codes that are available with their meanings, and
 #' what the different columns mean for them. 
+#' 
+#' The metadata table explains the meanings of data in
+#' the events table, which is retrieved through the
+#' exportData() function. It has three fields to describe
+#' what the general shape of the data is, while the other
+#' fields describe what data is stored in the field with that 
+#' name in the events table.
 #'
-#' This functions returns a loaded into memory R table,
+#' * Code_name is the XML code of this type of data
```

Comment:
Is that actually a capital C? Would be nice to be consistent with the other column names if it is.